sashimi-webapp/package.json

```diff
@@ -45,6 +45,7 @@
     "express": "^4.14.1",
     "extract-text-webpack-plugin": "^2.0.0-rc.2",
     "file-loader": "^0.10.0",
+    "flowchart.js": "^1.6.5",
```

Comment:
flowchart.js, mermaid, viz.js should be removed since we are using it directly in the script tag